content/articles/changing-plans.md

```diff
@@ -7,47 +7,71 @@ categories:
 
-There are two different types of plan changes based on when they are billed.
+<info>
+We do not offer refunds for plan changes. If you upgrade your plan, you will receive a proration credit for the time remaining on your previous plan. If you downgrade, we wait until the end of the billing cycle, then charge you for the new subscription.
+</info>
 
-## Pre-renewal change plans
+## Upgrading your plan
 
-These plan changes are billed immediately.
+If you **upgrade** your subscription before your monthly renewal, **the changes take effect and are billed immediately**.
 
-You will receive credit for all the unused time of your current subscription plan. This credit doesn't include extra items like email forwards or active zones. These will be billed again as part of the plan change. Read more about [subscription proration](/articles/understanding-invoice).
+You will receive a prorated discount corresponding to the unused time in your current subscription plan. This credit doesn't include extra items like email forwards or active zones, as these are pay-per-use features. These will be billed again as part of the plan change.
 
-## On-renewal change plans
+Read more about [subscription proration](/articles/understanding-invoice/#proration).
 
-These plan changes are billed at the end of your current billing cycle along with your subscription renewal.
+## Downgrading your plan
+
+If you **downgrade** your subscription, **the changes will be billed at the end of your current billing cycle**, along with your subscription renewal. All changes take effect at the time of renewal.
+
+There is no proration credit for downgrades. We wait until the end of the billing cycle, then change your plan.
 
 You can cancel these plan changes in your account settings:
 
 ![Cancel plan change](/files/account-billing-cancel-plan-change.png)
 
-<div class="section-steps" markdown="1">
-##### Changing your subscription plan
+
+## Changing your subscription plan
 
 1.  Log in to DNSimple with your user credentials.
-1.  [Find the account](https://dnsimple.com/user) from the account list. Click on the account to enter the account page.
-1.  Click the <label>Change plan</label> link under your plan section.
+2.  [Find the account](https://dnsimple.com/user) from the account list. Click on the account to enter the account page.
+3.  Click the <label>Change plan</label> link under your plan section.
 
-    ![Change plan](/files/account-billing-change-plan-link.png)
+**New screenshot** 
 
-1.  Explore the features for each plan. Click the <label>Select</label> button of the plan that best suits your needs.
+4.  Explore the features for each plan. Click the <label>Select</label> button of the plan that best suits your needs.
 
-    ![Identify](/files/change-plan-2.png)
+**new screenshot** 
```

Comment:
https://drive.google.com/drive/u/3/folders/1gq_IxXbuszRRYWBWK-lOujG2zxl9oRnw 

New screenshot that doesn't indicate the teams plan is $99/mo